tests/test_basic.py

```diff
@@ -192,6 +192,42 @@ def test_default_list(tmpdir, runner, create):
     assert todo.summary == 'test default list'
 
 
+def test_default_due_is_not_specified(tmpdir, runner, create):
```

Comment:
I think that would require at least 2 `if` statements (one for special case where no `default_due` is specified, one for special case where result is `None` if `default_due` is 0). That would make the tests less readable. Do you think it's worth the effort?